foundations/introduction/join_the_odin_community.md

```diff
@@ -1,5 +1,6 @@
 ### Introduction
-Working and collaborating with other people is an important part of working as a web developer. Therefore, we at The Odin Project encourage you to participate in our online chat community, which we'll talk more about below. By joining we can all grow together and help each other learn web development. While you're at it, you can check out our [Facebook page](https://www.facebook.com/theodinproject/) and [follow us on Twitter](https://twitter.com/TheOdinProject) and on [Instagram](https://www.instagram.com/theodinproject/).
+Working and collaborating with other people is an important part of working as a web developer. Therefore, we at The Odin Project encourage you to participate in our online chat community, which we'll talk more about below. By joining we can all grow together and help each other learn web development. While you're at it, you can check out our [Facebook page](https://www.facebook.com/theodinproject/) and [follow us on Twitter](https://twitter.com/TheOdinProject) and catch up on
```

Comment:
We need a comma between the facebook clause and the twitter clause, rather than "and".

Reply:
Updated.